pino.js

```diff
@@ -338,6 +343,15 @@ function pino (opts, stream) {
     }
   }
 
+  const baseLog = typeof iopts.baseLog === 'object' ? iopts.baseLog : defaultBaseLog
```

Comment:
Good catch, thanks. Perhaps this rule should be integrated to linter? Or start using const and let instead elsewhere to change the base.
Could you please elaborate more on the mentioned grouping you'd like to see? Can't seem to find a clear example of that to follow.